src/views/Menu.js

```diff
@@ -0,0 +1,313 @@
+    width: 8.25em;
+  }
+
+  @media only screen and (min-width: ${BETWEEN_SMALL_DEVICES_TABLET_UP}px) {
+    height: 7.25em;
+    width: 7.25em;
+  }
+
+  @media only screen and (min-width: ${TABLET_PORTRAIT_UP}px) {
+    height: 8.25em;
+    width: 8.25em;
+  }
+`;
+
+const StyledMenuItemName = styled.span`
+  text-transform: uppercase;
+  font-size: 0.875rem;
+  text-align: center;
+  letter-spacing: 0.1em;
+  color: #152540;
+  line-height: 1.2em;
+  width: 10em;
+
+  @media only screen and (min-width: ${SMALL_DEVICES_LANDSCAPE_UP}px) {
+    font-size: 1rem;
+  }
+
+  @media only screen and (min-width: ${DESKTOP_UP}px) {
+    font-size: 1.25rem;
+  }
+`;
+
+const StyledFootnote = styled.div`
+  background-color: #152540;
+  color: rgba(246, 233, 203, 1);
+  font-size: 0.875rem;
+  letter-spacing: 0.1em;
+  text-transform: uppercase;
+  text-align: center;
+  margin: 0 auto;
+  padding: 0.4em 0.8em;
+  line-height: 1.2em;
+  max-width: 20em;
+
+  @media only screen and (min-width: ${POST_IPHONE6_PORTRAIT_UP}px) {
+    max-width: 24em;
+  }
+
+  @media only screen and (min-width: ${POST_IPHONE6_PLUS_PORTRAIT_UP}px) {
+    max-width: 26em;
+  }
+
+  @media only screen and (min-width: ${PHONE_LANDSCAPE_UP}px) {
+    max-width: 28em;
+  }
+
+  @media only screen and (min-width: ${SMALL_DEVICES_LANDSCAPE_UP}px) {
+    font-size: 1rem;
+    max-width: 30em;
+  }
+
+  @media only screen and (min-width: ${DESKTOP_UP}px) {
+    font-size: 1.25rem;
+  }
+`;
+
+const StyledLink = styled(Link)`
+  color: rgba(246, 233, 203, 1);
+  text-decoration: underline;
+
+  &:hover {
+    opacity: 0.8;
+    cursor: pointer;
+  }
+`;
+
+const menuItems = [
+  { name: "Whole Roasted Suckling Pig", url: "whole_roasted_suckling_pig" },
+  { name: "Sashimi Platter", url: "sashimi_platter" },
+  { name: "Chinese Cold Platter", url: "chinese_cold_platter" },
+  { name: "French Style Grilled Lobster", url: "french_style_grilled_lobster" },
+  { name: "Steamed Shrimp", url: "steamed_shrimp" },
+  {
+    name: "Stir Fried Sliced Conch and Clam",
+    url: "stir_fried_sliced_conch_and_clam"
+  },
+  {
+    name: "Braised Crab Meat & Fish Chowder",
+    url: "braised_crab_meat_and_fish_chowder"
+  },
+  { name: "Steamed King Crab", url: "steamed_king_crab" },
+  { name: "Abalone with Oyster Sauce", url: "abalone_with_oyster_sauce" },
+  {
+    name: "Crispy Chicken with Garlic Sauce",
+    url: "crispy_chicken_with_garlic_sauce"
+  },
+  { name: "Steamed Fresh Fish", url: "steamed_fresh_fish" },
+  { name: "Sweet Sticky Rice", url: "sweet_sticky_rice" },
+  { name: "E-Fu Noodle", url: "e-fu_noodle" },
+  { name: "Cake / Dessert", url: "cake_dessert" }
+];
+
+class Menu extends Component {
+  componentDidMount() {
+    const { handleIsBrandDark, handleIsNavbarDark } = this.props;
+    handleIsBrandDark(false);
+    handleIsNavbarDark(false);
+  }
+
+  render() {
+    return (
+      <StyledMain>
+        <Helmet>
+          <title>Megan and Ringo's Wedding - The Ceremony</title>
```

Comment:
The `<title />` should mention that the current page is the menu.

Reply:
The title only mentions the ceremony for the other pages listed under the dropdown.

Reply:
Ah, I see. We'll need to change those `<title />` values then since each of those dropdown items is associated to a unique route.